src/ir/cfg_builder.cpp

```
@@ -127,6 +128,121 @@ static bool has_fall(const Instruction& ins) {
+                                                 : "requires conformance group base32");
+        }
+    }
+    if (const auto p = std::get_if<Jmp>(&ins)) {
+        if (!supports(platform,
+                      p->cond ? (p->cond->is64 ? bpf_conformance_groups_t::base64 : bpf_conformance_groups_t::base32)
+                              : bpf_conformance_groups_t::base32)) {
+            return reject_capability((p->cond && p->cond->is64) ? "requires conformance group base64"
+                                                                : "requires conformance group base32");
+        }
+    }
+    if (const auto p = std::get_if<LoadPseudo>(&ins)) {
```

Clang (14.0.6)

[note] +1, including nesting penalty of 0, nesting level increased to 1

(clang)